src/views/components/SearchForm/component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Col, Input, Row } from 'antd'
+
+const SearchForm = () => (
+  <Row
+    justify="center"
+    gutter={{
+      xs: 8,
+      sm: 16,
+      md: 24,
+      lg: 22
+    }}
+  >
+    <Col
+      className="gutter-row"
+      xs={{ span: 20 }}
+      sm={{ span: 20 }}
+      md={{ span: 14 }}
+      lg={{ span: 12 }}
+      xl={{ span: 10 }}
+    >
+      <Input.Search
+        placeholder="Enter movie name"
+        size="large"
+        enterButton="Search"
+        className="top-margin"
+      />
```

Comment:
do you plan to implement i18n for this?

Reply:
Yes, after adding functionality for this component. In next PR, now search is not working